Content.Server/Atmos/Components/GasTankComponent.cs

```diff
@@ -283,7 +283,7 @@ public void CheckStatus()
                     if(environment != null)
                         atmosphereSystem.Merge(environment, Air);
 
-                    SoundSystem.Play(Filter.Pvs(Owner), _ruptureSound.GetSound(), Owner.Transform.Coordinates, AudioHelpers.WithVariation(0.125f));
```

Comment:
Given the entity is deleted in the next frame, it makes no sense to play this sound on the owner. Revert the change